src/Agent/NewRelic/Profiler/Profiler/ICorProfilerCallbackBase.h

```diff
@@ -752,7 +780,7 @@ namespace Profiler {
         {
             //This will pull the app pool name out of instances of IIS > 5.1
             //For more information see http://msdn.microsoft.com/en-us/library/ms524602(v=vs.90).aspx
-            auto appPoolId = systemCalls->TryGetEnvironmentVariable(_X("APP_POOL_ID"));
+            auto appPoolId = systemCalls->GetAppPoolId();
```

Comment:
It's going to be really helpful moving forward to centralize all places where configuration is pulled from environment variables into one file. 💯